api/v1alpha1/gatekeeper_types.go

```diff
@@ -62,17 +62,17 @@ type ImageConfig struct {
 type AuditConfig struct {
 	// +kubebuilder:validation:Minimum:=0
 	// +optional
-	Replicas *int64 `json:"replicas,omitempty"`
+	Replicas *int32 `json:"replicas,omitempty"`
 	// +optional
 	AuditInterval *metav1.Duration `json:"auditInterval,omitempty"`
 	// +kubebuilder:validation:Minimum:=0
```

Comment:
NOTE: JSON does not support unsigned integers so without the validation marker the user can input an integer < 0 and when you unmarshall the JSON for that integer to a go struct field that is unsigned you would get the unsigned representation of that integer e.g. MAX_INT on the `uint64` type (2^64-1) when the user inputs `-1`. So better to leave the validation marker.